### ISSUE TITLE
error occurs when converting NuScenes data

### ISSUE THREAD

**Issue by zuo1188** [hustvl/VAD#4]
tools/data_converter/vad_nuscenes_converter.py
line 221
shoud be fixed to this?
```
for dic in nusc.category:
        cat2idx[dic['name']] = dic['index']

----->
for idx,dic in enumerate(nusc.category):
        cat2idx[dic['name']] = idx
```

**Comment by rb93dett** [comment]
Can you provide your running command and the error information? I don't have a problem here when converting data.

**Comment by zuo1188** [comment]
command:
`python tools/data_converter/vad_nuscenes_converter.py nuscenes --root-path ./data/nuscenes --out-dir ./data/nuscenes --extra-tag vad_nuscenes --version v1.0 --canbus ./data`
result:
```
Traceback (most recent call last):
  File "tools/data_converter/vad_nuscenes_converter.py", line 979, in <module>
    nuscenes_data_prep(
  File "tools/data_converter/vad_nuscenes_converter.py", line 937, in nuscenes_data_prep
    create_nuscenes_infos(
  File "tools/data_converter/vad_nuscenes_converter.py", line 106, in create_nuscenes_infos
    train_nusc_infos, val_nusc_infos = _fill_trainval_infos(
  File "tools/data_converter/vad_nuscenes_converter.py", line 222, in _fill_trainval_infos
    cat2idx[dic['name']] = dic['index']
KeyError: 'index'
```

**Comment by rb93dett** [comment]
Do you install nuscenes-devkit package in your python environment?
Besides, you can directly download our pre-generated nuscenes data following this [README](https://github.com/hustvl/VAD/blob/main/docs/prepare_dataset.md).

**Comment by zuo1188** [comment]
> Do you install nuscenes-devkit package in your python environment? Besides, you can directly download our pre-generated nuscenes data following this [README](https://github.com/hustvl/VAD/blob/main/docs/prepare_dataset.md).
1.My pip list:
nuscenes-devkit          1.1.10
2.OK

**Comment by rb93dett** [comment]
I don't seem to have this problem in line 221, my nuscenes-devkit version is 1.1.9, You can try version 1.1.9 to see if this problem can be solved.

**Comment by zuo1188** [comment]
> 
fixed

**Comment by zuo1188** [comment]
> 
fixed

**Comment by alfredgu001324** [comment]
@zuo1188 Hi, I am having the same problem and I have devkit==1.1.9. How did you fix it at the end?

**Comment by chenyonx** [comment]
hi, the fixed way is to update nuscenes-devkit 1.1.10?

**Comment by zuo1188** [comment]
> 

yes

**Comment by zuo1188** [comment]
> hi, the fixed way is to update nuscenes-devkit 1.1.10?

downgrade to 1.1.9
